assets/src/components/routeLadder.tsx

```diff
@@ -130,7 +130,14 @@ const RouteLadder = ({
   }
 
   const byPosition: VehiclesByPosition = groupByPosition(
-    vehiclesAndGhosts,
+    vehiclesAndGhosts?.filter(
+      (vehicleOrGhost) =>
+        !(
```

Comment:
@lemald the `!` gets visually lost here and makes the function hard to understand, introducing an intermediate variable and negating that instead would help readability.

Reply:
Updated.